mobile/lib/ui/viewer/file/zoomable_image.dart

```diff
@@ -239,7 +242,10 @@ class _ZoomableImageState extends State<ZoomableImage> {
             _isGIF(), // since on iOS GIFs playback only when origin-files are loaded
       ).then((file) {
         if (file != null && file.existsSync()) {
-          _onFinalImageLoaded(Image.file(file).image);
+          _onFinalImageLoaded(
```

Comment:
`_onFinalImageLoaded` can create the `ImageProvider` with the `file`, so we could reduce one parameter? It would then make sense to rename the method to `_onFileLoaded`.

Reply:
Have made these changes.